src/AltinnCore/Common/Languages/ini/en.ini

```diff
@@ -8,9 +8,11 @@ choose = Choose
 choose_label = Choose label
 choose_method = Choose method
 close = Close
+contains = Contains
 control_submit = Control and submit
 create = Create
 create_new = Create new
+customer_service_phone_number = 67 00 60 00
```

Comment:
Maybe a landcode?

Reply:
I squashed and merged before I saw this. I can add it to my next branch.